src/App.css

```diff
@@ -1,24 +1,83 @@
-.App {
+.Title {
+  display:flex;
+  flex-direction: column;
+  align-items: stretch;
   text-align: center;
+  background-color: #00ffff;
+  padding-top:80px;
+  padding-bottom: 0px;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 80px;
+.App-intro {
+  font-size: 600%;
+  text-align: center;
+  margin-top: 0;
+  background-color: #00ffff;
+  color: #662e9e;
+}
+
+.Instructions{
+  display:flex;
+  flex-direction: column;
+  align-items: stretch;
+  justify-content: center;
+  background-color: #00ffff;
+  height:550px;
+  padding-bottom:20px;
 }
 
-.App-header {
-  background-color: #222;
-  height: 150px;
+.Instructions-header{
+  margin-top:0;
+  margin-bottom:0;
+  text-align: center;
+  background-color: #00ffff;
+  color: #662e9e;
+  padding: 40px;
+  font-size: 140%;
+  font-family: Times;
+}
+
+.Instructions-body{
+  background-color: #00ffff;
+  flex-direction:column;
+  align-items:center;
   padding: 20px;
-  color: white;
+  height: 200px;
+  font-size: 100%;
+  font-family: "Trebuchet MS";
```

Comment:
Let's make the font the same everywhere.

Reply:
If you want to pick a cooler font that's awesome, here's some we can check out: https://fonts.google.com/

Reply:
This might help: https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-images-fonts-and-files